src/Libraries/RevitNodes/Elements/Views/Sheet.cs

```diff
@@ -506,6 +521,110 @@ public static Sheet ByNameNumberTitleBlock(string sheetName, string sheetNumber,
             return new Sheet(sheetName, sheetNumber, titleBlockFamilyType.InternalFamilySymbol);
         }
 
+        /// <summary>
+        /// Duplicates A Sheet. 
+        /// </summary>
+        /// <param name="sheet">The Sheet to be Duplicated</param>
+        /// <param name="viewDuplicateOption">Enter View Duplicate Option: 0 = Duplicate. 1 = AsDependent. 2 = WithDetailing.</param>
+        /// <param name="prefix"></param>
+        /// <param name="suffix"></param>
+        /// <returns></returns>
+        public static Sheet DuplicateSheet(Sheet sheet, int viewDuplicateOption = 0, string prefix = "", string suffix = "")
+        {            
+            if (sheet == null)
+                throw new ArgumentNullException(nameof(sheet));
+
+            ViewDuplicateOption Option = 0;
+            switch (viewDuplicateOption)
```

Comment:
Need to check with QA if we need the option for Sheet, or using `ViewDuplicateOption.WithDetailing` by default.